modules/pam_env/pam_env.8.xml

```diff
@@ -52,13 +52,26 @@
       variables as well as <emphasis>PAM_ITEM</emphasis>s such as
       <emphasis>PAM_RHOST</emphasis>.
     </para>
-    <para>
+    <para condition="with_vendordir">
+      By default rules for (un)setting of variables are taken from the
+      config file <filename>/usr/etc/security/pam_env.conf</filename>. An
+      alternate file is <filename>/usr/etc/security/pam_env.conf</filename>
+      or can be specified with the <emphasis>conffile</emphasis> option.
+    </para>
+    <para condition="without_vendordir">
       By default rules for (un)setting of variables are taken from the
       config file <filename>/etc/security/pam_env.conf</filename>. An
       alternate file can be specified with the <emphasis>conffile</emphasis>
       option.
     </para>
-    <para>
+    <para condition="with_vendordir">
+      Second a file (<filename>/usr/etc/environment</filename> by default) with simple
+      <emphasis>KEY=VAL</emphasis> pairs on separate lines will be read.
+      An alternate file is <filename>/etc/security/pam_env.conf</filename>
+      or can be specifed with the <emphasis>envfile</emphasis> option.
+      And with the <emphasis>readenv</emphasis> option this can be completely disabled.
+    </para>
```

Comment:
Please also document /etc/security/pam_env.conf.d for the case we use libeconf, so that people know about all new features.

Reply:
I have updated the docu.